docs/quickstarts/insights-malware/insights-malware.yml

```diff
@@ -10,7 +10,7 @@ spec:
     color: orange
   displayName: Find and report malware 
   icon: data:image/svg+xml;base64,PHN2ZyBpZD0iZTA1M2ZiYWQtY2I0Ny00OGNiLThiNDMtZDczY2RmMzQyNGJjIiBkYXRhLW5hbWU9IkxheWVyIDEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmlld0JveD0iMCAwIDM3IDM3Ij4KICA8ZGVmcz4KICAgIDxzdHlsZT4KICAgICAgLmE0NjBjMWU3LWMyNjAtNDc5MS1iZjc0LTI0ZmE4OGU4MTcwZSB7CiAgICAgICAgZmlsbDogI2UwMDsKICAgICAgfQoKICAgICAgLmU4YTQ2MmQ1LTgzYWUtNGRjMi04OTVlLWE3NDU3ZjY2MDNjYSB7CiAgICAgICAgZmlsbDogI2ZmZjsKICAgICAgfQogICAgPC9zdHlsZT4KICA8L2RlZnM+CiAgPHBhdGggZD0iTTI3LjUuNUg5LjVhOSw5LDAsMCwwLTksOXYxOGE5LDksMCwwLDAsOSw5aDE4YTksOSwwLDAsMCw5LTlWOS41YTksOSwwLDAsMC05LTlaIi8+CiAgPHBhdGggY2xhc3M9ImE0NjBjMWU3LWMyNjAtNDc5MS1iZjc0LTI0ZmE4OGU4MTcwZSIgZD0iTTI2LjUsMjcuMTI1aC02YS42MjUuNjI1LDAsMCwxLS42MjUtLjYyNXYtNmEuNjI1LjYyNSwwLDAsMSwuNjI1LS42MjVoNmEuNjI1LjYyNSwwLDAsMSwuNjI1LjYyNXY2QS42MjUuNjI1LDAsMCwxLDI2LjUsMjcuMTI1Wm0tNS4zNzUtMS4yNWg0Ljc1di00Ljc1aC00Ljc1WiIvPgogIDxwYXRoIGNsYXNzPSJhNDYwYzFlNy1jMjYwLTQ3OTEtYmY3NC0yNGZhODhlODE3MGUiIGQ9Ik0yNi41LDE3LjEyNWgtNmEuNjI1LjYyNSwwLDAsMS0uNjI1LS42MjV2LTZhLjYyNS42MjUsMCwwLDEsLjYyNS0uNjI1aDZhLjYyNS42MjUsMCwwLDEsLjYyNS42MjV2NkEuNjI1LjYyNSwwLDAsMSwyNi41LDE3LjEyNVptLTUuMzc1LTEuMjVoNC43NXYtNC43NWgtNC43NVoiLz4KICA8cGF0aCBjbGFzcz0iYTQ2MGMxZTctYzI2MC00NzkxLWJmNzQtMjRmYTg4ZTgxNzBlIiBkPSJNMTYuNSwxNy4xMjVoLTZhLjYyNS42MjUsMCwwLDEtLjYyNS0uNjI1di02YS42MjUuNjI1LDAsMCwxLC42MjUtLjYyNWg2YS42MjUuNjI1LDAsMCwxLC42MjUuNjI1djZBLjYyNS42MjUsMCwwLDEsMTYuNSwxNy4xMjVabS01LjM3NS0xLjI1aDQuNzV2LTQuNzVoLTQuNzVaIi8+CiAgPHBhdGggY2xhc3M9ImU4YTQ2MmQ1LTgzYWUtNGRjMi04OTVlLWE3NDU3ZjY2MDNjYSIgZD0iTTE2LjUsMTkuODc1aC01YS42MjUuNjI1LDAsMCwwLDAsMS4yNWgzLjQ5MWwtNC45MzMsNC45MzNhLjYyNS42MjUsMCwwLDAsLjg4NC44ODRsNC45MzMtNC45MzNWMjUuNWEuNjI1LjYyNSwwLDAsMCwxLjI1LDB2LTVBLjYyNS42MjUsMCwwLDAsMTYuNSwxOS44NzVaIi8+Cjwvc3ZnPgo=
-  description: Scan, detect, monitor, and report malware attacks on systems within a RHEL environment. Utilizes YARA-pattern-matching software and malware-detection signatures.
-  link:
+  description: Use the malware detection service to know when systems in your RHEL infrastructure are exposed to risks and attacks.
+link:
```

Comment:
@michelle-purcell was this indentation change also intentional? It's not breaking the build - but I'm not sure the link will render as we expect.